components/insight/SRC/org/openmicroscopy/shoola/agents/measurement/view/MeasurementTableModel.java

```diff
@@ -188,12 +164,54 @@ public Object getValueAt(int row, int col)
                     }
                 }
                 return buffer.toString();
-            } else if (value instanceof Number) {
-                double d = ((Number) value).doubleValue();
-                return UIUtilities.twoDecimalPlacesAsNumber(d);
+            } else {
+                if (value instanceof Number) {
```

Comment:
shouldn't this if and the next just continue the enclosing "else if"...? (similar below)